### PR TITLE
wip Docker deployments

### PR DESCRIPTION
**Pull request by sirupsen**

### CODE REVIEW

File: script/run [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# create /tmp
+# port 4000
+
+if [[ $1 == "web" ]]; then
```
Comment (bouk) [inline script/run:3]: That's a neat way of doing it!

File: README.md [hunk 2]
```diff
@@ -73,5 +73,41 @@ forwarded from the Vagrant-managed VM.
...
+Run the Contestrus worker:
+
+```bash
+docker run \
+  --volume  /var/log/contestrus:/app/log \
+  --volume  /var/lib/contestrus:/db \
+  --volume  /var/run/docker.sock:/var/run/docker.sock \
+  --volume  /var/lib/docker:/var/lib/docker \
+  --env     RAILS_ENV=production \
+  --publish 4000:80 \
```
Comment (bouk) [inline README.md]: I don't think this should be here

Comment (sirupsen) [inline README.md]: Ya, could probably just listen straight on 80 with Puma and expose 80.

Comment (bouk) [inline README.md]: I meant that the worker doesn't need to listen on port 80

Comment (sirupsen) [inline README.md]: oh yeah

**Pull request merged**